src/compute-engine/boxed-expression/rules.ts

```diff
@@ -715,12 +728,11 @@ export function applyRule(
 
   if (canonical && match) {
     const awc = getWildcards(match);
-    const originalMatch = match;
-    match = match.canonical;
```

Comment:
This (line) appeared to be non-effective, its removal not affecting any tests, and so removed it.